src/Network/Plugin.cs

```diff
@@ -681,8 +682,11 @@ private static bool UnblockFile(string fileName)
             return DeleteFile(fileName + ":Zone.Identifier");
         }
 
-        private void HandleCmdList
-            (GraphicsDevice device, IntPtr ptr, int length, Dictionary<IntPtr, GraphicsResource> resources)
+        private void HandleCmdList(
+            GraphicsDevice device,
+            IntPtr ptr,
+            int length,
+            IDictionary<IntPtr, GraphicsResource> resources)
```

Comment:
Why IDictionary and not a Dictionary class directly?

Reply:
that way other `IDictionary<TKey, TValue>` impls can be passed in. SortedDict/ConcurrentDict/MyFancyDict....